README.md

```diff
@@ -16,3 +16,18 @@ https://github.com/equinor/webviz-ert
 # From the downloaded project's root folder - to install
 pip install .
 ```
+
+## Run Webviz-ert
+Webviz-ert connects automatically to a storage server running in [ERT](https://github.com/equinor/ert).
+Here are a few steps to get an example version of webviz-ert running.
+
+```sh
+# Run simulations in ert - <ert-root-folder>/test-data/local/poly_example/
+ert ensemble-smooter --target-case smoother poly.ert
+
```

Comment:
should It be stated they have to wait until the ert run is done? I guess ert api while running is not going to fly?

Reply:
fair point